### PR TITLE
Feature/custom calendar

### PR DESCRIPTION
**Pull request by CibelePaulinoAndrade**
Custom Display finished 
- Calendar
- How many times
- Frequency (Weekly, daily or monthly)

### CODE REVIEW

File: FinalChallenge/Enums/DisplaytypeEnum.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    var displayItens: [String] {
+        switch self {
+        case .weekDay:
+            return ["M", "T", "W", "T", "F", "S", "S"]
+        case .howManyTimes:
+            return ["1x", "2x", "3x", "4x"]
+        case .frequency:
+            return ["Daily", "Weekly", "Monthly"]
+        }
+    }
```
Comment (jessicalewinter) [inline FinalChallenge/Enums/DisplaytypeEnum.swift]: It's interesting to add NSLocalizedString later

File: FinalChallenge/Views/Rounded Display/NeonRoundView.swift [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        if animating {
+            self.superview?.layoutIfNeeded()
+        }
+            
+        self.centerXConstraint = self.centerXAnchor.constraint(equalTo: view.centerXAnchor)
+        self.centerYConstraint = self.centerYAnchor.constraint(equalTo: view.centerYAnchor)
+        
+        self.centerYConstraint?.isActive = true
+        self.centerXConstraint?.isActive = true
+
```
Comment (jessicalewinter) [inline FinalChallenge/Views/Rounded Display/NeonRoundView.swift]: Use NSLayoutConstraint.activate([]) to maintain a pattern in the project

File: FinalChallenge/Enums/DisplaytypeEnum.swift (hunk 1 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Enums/DisplaytypeEnum.swift]: Será feito junto com o restante do app

File: FinalChallenge/Views/Rounded Display/NeonRoundView.swift (hunk 2 above)
Comment (CibelePaulinoAndrade) [reply FinalChallenge/Views/Rounded Display/NeonRoundView.swift]: Adicionado ao bugfix

**Pull request merged**